pkg/resolution/resolver/git/resolver.go

```diff
@@ -157,7 +159,7 @@ func validateRepoURL(url string) bool {
 	return re.MatchString(url)
 }
 
-func ResolveAnonymousGit(ctx context.Context, params map[string]string) (framework.ResolvedResource, error) {
+func ResolveAnonymousGit(ctx context.Context, params map[string]string, kubeclient kubernetes.Interface, logger *zap.SugaredLogger, cache *cache.LRUExpireCache, ttl time.Duration) (framework.ResolvedResource, error) {
```

Comment:
Nitpick: this is no longer necessarily `AnonymousGit`, since it supports authentication.